content/post/078-go-ipfs-0.4.23.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2020-01-27
+url: 2020-01-27-go-ipfs-0-4-23
+title: go-ipfs 0.4.23 released
+author: Alan Shaw
+---
+
+> Would sir/madam care for another patch release while they wait?
+
+Yes that's right, the next feature release of go-ipfs (0.5.0) is, well, running a tiny bit behind schedule. In the mean time though we have patches, and I'm not talking pirate eye patches, I'm talking bug fixes. We're hunting these bugs like they're Pokemon, and jeez, do we come across some rare and difficult to fix ones? - you betcha.
+
+Alright, enough funny business, what's the deal? Ok so, I don't want to _alarm_ anyone but this release has some **critical fixes** and if you're using go-ipfs or know someone who is then you and your friends need to slide into your upgrade pants and give those IPFS nodes a good wipe down ASAP.
+
+## 🔦 Highlights
+
+### 🤝 Fixed Spontaneous TLS Disconnects
+
+If this isn't reason enough to upgrade I don't know what is. Turns out, a TLS handshake _may_ have accidentially been unintentionally aborted for no good reason 😱. I shouldn't need to tell you this but \*whispers\* that's really bad for connectivity! Excellent news though, it's fixed, and if you want to stay well connected in the glorious future of p2p communications then you should definitely upgrade.
```

Comment:
Let's mention:

1. This is a rare race condition.
2. Maybe play down the "really bad" part. I get that it's a joke but people might interpret this to mean that this bug was frequently causing random disconnects.
3. We should probably mention that TLS is still experimental and go-ipfs still uses SECIO by default unless configured otherwise.

Reply:
(and the reason this is critical is that TLS will be enabled by default in 0.5.0)